src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -0,0 +1,142 @@
+package life.qbic.datamodel.datasets
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+import life.qbic.datamodel.datasets.datastructure.folders.DataFolder
+import life.qbic.datamodel.datasets.datastructure.folders.nanopore.*
+
+/**
+ * A dataset that represents a Oxford Nanopore Measurement.
+ *
+ * @author: Sven Fillinger
+ */
+final class OxfordNanoporeMeasurement {
+
+    private final Map<String, DataFolder> folders
+
+    private final List<DataFile> logFiles
+
+    private final MeasurementFolder measurementFolder
+
+    private final boolean pooledMeasurement
+
+    protected OxfordNanoporeMeasurement(String name, String path, List children) {
+        logFiles = new ArrayList<>()
+        folders = new HashMap<>()
+
+        this.measurementFolder = MeasurementFolder.create(name, path, children)
+
+        createContent()
+
+        pooledMeasurement = folders["fast5passed"] ? folders["fast5passed"].getTheChildren().get(0) instanceof Fast5Folder : false
```

Comment:
I do not understand this statement. You check if the `fast5_pass` folder contains any other Fast5Folder.class instances. But should you not rather look for some folder with a QBiC barcode?

Also in case the sequencing went bad you might happen to have only results in the `fast5_fail` folder for low PHRED scores.

Reply:
According to the JSON schema, there are only two types of children a fast5passed folder can contain: a Fast5File or a Fast5Folder. The latter is a subtype of a barcoded folder. If we have a barcoded folder here, we know it must be a pooled measurement.

I agree to the second statement: worst case scenario is, that all samples have failed.